### ISSUE TITLE
Support moving between *sections* of the "themes" menu

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [microsoft/vscode#70861]
Issue Type: <b>Feature Request</b>

On macOS, search fields often have ‘sections:’

<img width="792" alt="Screen Shot 2019-03-20 at 16 05 36" src="https://user-images.githubusercontent.com/200/54719102-0a98b380-4b2a-11e9-9f50-a67d0e5733a2.png">

More importantly, in all such sections across the operating system, there's a consistent set of keybindings for *navigating* these sections. For instance, in the above interface, as an element within the `Other` section is highlighted, hitting <kbd>⌘</kbd><kbd>↓</kbd> would ‘leap’ to the first item of the `Folders` section, regardless of *which* element in `Other` was currently selected.

Meanwhile, in VScode, there are several such interfaces. Notably, for me, is the "Themes" interface:

<img width="623" alt="Screen Shot 2019-03-20 at 16 10 28" src="https://user-images.githubusercontent.com/200/54719378-b510d680-4b2a-11e9-9746-4a92726c1c97.png">

Unfortunately, if my cursor is ‘high up’ in the `Light themes` section, moving to the `Dark themes` section is very costly to my eyes, and not quick at all — in fact, it's one of the *only* things I reliably use my mouse for, every single day!

tl;dr: I'd love keybinds in the `list.*` category to jump to the first item in the (previous/next) section to the one that the cursor is currently inside. `(=`

VS Code version: Code 1.32.3 (a3db5be9b5c6ba46bb7555ec5d60178ecc2eaae4, 2019-03-14T22:46:48.547Z)
OS version: Darwin x64 18.2.0
